### PR TITLE
Add setname

### PR DESCRIPTION
**Pull request by csconner1998**

### CODE REVIEW

File: bot.py [hunk 1]
```diff
@@ -184,6 +189,26 @@ async def match_history(ctx, count: int):
     await db_handler.get_match_history(ctx=ctx, count=count)
     await res.delete_original_message()
 
+# Set players nickname with Summoner Name
+@bot.slash_command(guild_ids=[test_guild_id])
```
Comment (GNMoseke) [inline bot.py]: nit: shouldn't need `guild_ids` here b/c bot is initialized with `test_guild_id` already

File: bot.py [hunk 2]
```diff
@@ -184,6 +189,26 @@ async def match_history(ctx, count: int):
...
+async def setname(ctx, summoner_name: str):
+    if ctx.channel_id != name_assign_channel:
+        owner = ctx.author
+        direct_message = await owner.create_dm()
+        await direct_message.send("Can only use /setname in #name-assign. If you need to change your name, please reach out to a Staff member")
```
Comment (GNMoseke) [inline bot.py]: nit: I think you can get away with just
```python
await ctx.author.send("message")
```
but if you split it like this on purpose nw

File: bot.py [hunk 3]
```diff
@@ -184,6 +189,26 @@ async def match_history(ctx, count: int):
...
+    try:
+        role = discord.utils.get(ctx.guild.roles, name="Member")
+        sum = watcher.summoner.by_name(my_region,summoner_name)
+        await ctx.author.add_roles(role)
+        await ctx.author.edit(nick=sum["name"])
+        await ctx.respond("Welcome " + sum["name"])
+    except ApiError as e:
+        code = e.response.status_code
+        if code == 401 or code == 403:
+            await ctx.respond("<@&"+bot_dev_role+"> needs to update riot API key. Please reachout to Staff to fix.")
```
Comment (GNMoseke) [inline bot.py:207]: oh this is smart idea. I should start doing this

File: inhouse/command_handlers/match.py [hunk 4]
```diff
@@ -104,7 +113,31 @@ async def send_match_description(self):
...
         self.match_description_message = match_desc
-
+    async def send_channel(self, member: discord.Member, channel: discord.VoiceChannel):
+        try:
+            await member.move_to(channel)
+            return ""
+        except Exception as e:
+            print(e)
+            return "<@" + member.id + "> join <#" + channel.id + ">\n"
+    async def move_to_channels(self, ctx: discord.context.ApplicationContext):
```
Comment (GNMoseke) [inline inhouse/command_handlers/match.py]: nit: can you put a newline here?

**Review by GNMoseke — APPROVED** [review]

File: bot.py (hunk 3 above)
Comment (csconner1998) [reply bot.py:207]: Error handling pog

**Review by GNMoseke — APPROVED** [review]

**Pull request merged**